Resources/public/js/views/ez-usermenuitemfireeventview.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuitemfireeventview', function (Y) {
+    "use strict";
+    /**
+     * Provides the user menu item fire event view
+     *
+     * @module ez-usermenuitemfire event view
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The user menu item fire event view. 
+     *
+     * @namespace eZ
+     * @class UserMenuItemFireEventView
+     * @constructor
+     * @extends Y.eZ.UserMenuItemView
+     */
+    Y.eZ.UserMenuItemFireEventView = Y.Base.create('UserMenuItemFireEventView', Y.eZ.UserMenuItemView, [], {
+        events: {
+            '.ez-view-usermenuitemfireeventview': {
+                'tap': '_logOut'
+            }
+        },
+
+        initializer: function () {
+            this.containerTemplate = '<li class="' + this._generateViewClassName(this._getName()) + '"/>';
+        },
+
+        /**
+        * Renders the item view
+        *
+        * @method render
+        * @return {eZ.UserMenuItemFireEventView} the view itself
+        */
+        render: function () {
+            this.get('container').setHTML(this.template(this.getAttrs()));
+            return this;
+        },
+
+        /**
+         * Tap event handler on the logout link
+         *
+         * @method _logOut
+         * @protected
+         * @param event {Object} tap event facade
+         */
+        _logOut: function (event) {
```

Comment:
it should not be called `_logOut` as it's much more generic now. `_fireEvent` maybe ? Also the js doc should be updated.